SmartPlaces.Facilities/lib/OntologyMapper/test/OntologyMapper.Test.csproj

```diff
@@ -16,16 +16,19 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="coverlet.collector" Version="3.1.0" />
+		<PackageReference Include="coverlet.collector" Version="3.1.2">
+		  <PrivateAssets>all</PrivateAssets>
+		  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
 		<PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
-		<PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.8" />
+		<PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.9" />
```

Comment:
Since this is being upgraded does is the Newtonsoft.Json still needed? If so, I think it should say Include, not Update

Reply:
It still needs newtonsoft. 10.0.3